design/manual-authn-and-authz/AUTHENTICATION.md

```diff
@@ -0,0 +1,116 @@
+# Manual Authentication
+
+## Requiring authentication in Kafka
+
+To make sure the clients are always authenticated the Kafka brokers need to be configured to enforce TLS client authentication on the `clienttls` listener.
+To do so, the option `listener.name.clienttls.ssl.client.auth` needs to be set to `required`.
+It can be set in the `Kafka` resource:
+
+```
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    ...
+    config:
+      listener.name.clienttls.ssl.client.auth: required
+    ...
+  zookeeper:
+    ...
+```
+
+## Downloading CA private and public keys
+
+The public and private keys for the clients Certification authority (CA) can be downloaded from the `Secret` created by the Cluster Operator (CO).
+The `Secret` is names `<cluster-name>-clients-ca` where the `<cluster-name>` has to be replaced by the name of your cluster (for example `my-cluster-clients-ca`).
+Following commands can be used to download both keys _(adapt the secret name according to your actual configuration)_:
+
+```
+oc get secret my-cluster-clients-ca -o jsonpath='{.data.clients-ca\.crt}' | base64 -d > clients-ca.crt
+oc get secret my-cluster-clients-ca -o jsonpath='{.data.clients-ca\.key}' | base64 -d > clients-ca.key
+```
+
+These keys will be used to sign the user certificates.
+The private key should be kept secret as it can be used to generate another user certificates.
+
+## Signing user certificates
+
+For the manual authentication process, the end-user has to download the client CA certificate and use it to sign user certificates.
+That can be done using many different tools.
+This document shows example of how to do it using [CFSSL](https://github.com/cloudflare/cfssl).
```

Comment:
Yeah ... I guess some might prefer `openssl` because it is often already installed (I do not think CFSSL is seen as untrustworthy, just less known). However:
* last time I wrote such guide for OpenSSL it was more complicated and we do not want to invest too much into a temporary document for temporary solution 
* there is plenty of OpenSSL private CA guides on the internet, so I think it is ok not to have it here.